website/static/templates.json

```diff
@@ -3,2021 +3,2558 @@
     "title": "Starter - Bicep",
     "description": "A starter template with Bicep as infrastructure provider",
     "preview": "./templates/images/test.png",
-    "website": "https://github.com/Azure/azure-dev",
+    "authorUrl": "https://github.com/Azure/azure-dev",
     "author": "Azure Dev",
     "source": "https://github.com/Azure-Samples/azd-starter-bicep",
     "tags": [
       "bicep",
       "msft"
-    ]
+    ],
+    "id": "99c380c4-915a-41b2-9517-45ed63d7ef7f"
   },
   {
     "title": "Starter - Terraform",
     "description": "A starter template with Terraform as infrastructure provider",
     "preview": "./templates/images/test.png",
-    "website": "https://github.com/Azure/azure-dev",
+    "authorUrl": "https://github.com/Azure/azure-dev",
     "author": "Azure Dev",
     "source": "https://github.com/Azure-Samples/azd-starter-terraform",
     "tags": [
       "terraform",
       "msft"
-    ]
+    ],
+    "id": "23ccf0c3-8850-4593-805e-6a0fd42c1093"
   },
   {
     "title": "WordPress with Azure Container Apps",
     "description": "A blueprint to easily and quickly create and deploy your first scalable and secure WordPress site to Azure, leveraging Azure Container Apps with Azure Database for MariaDb.",
     "preview": "./templates/images/apptemplate-wordpress-on-ACA.png",
-    "website": "https://github.com/kpantos",
+    "authorUrl": "https://github.com/kpantos",
     "author": "Konstantinos Pantos",
     "source": "https://github.com/Azure-Samples/apptemplate-wordpress-on-ACA",
     "tags": [
-      "php",
-      "javascript",
+      "bicep",
+      "msft"
+    ],
+    "azure_service": [
```

Comment:
Consider using camel casing similar to `authorUrl`. This field should be updated to `azureService`.